src/Controller.ts

```diff
@@ -35,22 +43,35 @@ export const Controller = {
 		return manager;
 	},
 
-	getEnabledEngines: async (): Promise<RuleEngine[]> => {
+	getAllEngines: async (): Promise<RuleEngine[]> => {
 		const engines: RuleEngine[] = [];
-		const logger: Logger = await Logger.child('Controller');
-
 		for (const engine of container.resolveAll<RuleEngine>(Services.RuleEngine)) {
```

Comment:
I removed the throw from this method, controller#resolveAll will never return a 0 length array. It throws it's own exception that indicates a programming error.